modules/01-filter.js

```diff
@@ -313,7 +312,7 @@ async function processCardAction(interaction) {
         break;
       case "modCardMajor":
         infraction.value = 5;
-        embed.addField("Resolved", `${mod.displayname} issued a 5 point warning.`);
+        embed.addField("Resolved", `${mod.displayName} issued a 5 point warning.`);
```

Comment:
I suggest escaping the display name:
```suggestion
        embed.addField("Resolved", `${u.escapeText(mod.displayName)} issued a 5 point warning.`);
```